crates/telio-model/src/mesh.rs

```diff
@@ -124,7 +124,10 @@ impl From<&PeerBase> for Node {
         Self {
             identifier: peer.identifier.clone(),
             public_key: peer.public_key,
-            nickname: peer.nickname.clone(),
+            nickname: peer
```

Comment:
Did these really need to be changed? I think `Option<T>` is `Clone` if `T` is `Clone`, which `Hidden` is, so just cloning should be fine. Though I'm guessing you got compiler errors for it?

Reply:
`PeerBase.nickname` is `Option<Hidden<String>>`, while `Node.nickname` is `Option<String>` so the change was to unwrap the the inner `String` from `Hidden<T>`.

Reply:
Right, makes sense!